docs/toolbox/accessibility_indicators/pt_trip_count.md

```diff
@@ -9,12 +9,21 @@ This indicator displays the **average number of public transport departures** pe
 
 ## 1. Explanation
 
-The Public Transport (PT) Trip Count shows the **average number of public transport departures per hour** for a selected **time interval** for each public transport stop on a point layer. The results can be visualized either as a sum of all transport modes or by focussing on one dedicated mode (e.g. bus, tram, metro, rail).
+The Public Transport (PT) Trip Count shows the **average number of public transport departures per hour** for a selected **time interval** for each public transport stop on a point layer. The results can be visualized either as a sum of all transport modes or by focusing on one dedicated mode (e.g. bus, tram, metro, rail).
```

Comment:
It's a bit unclear what "... each public transport stop on a point layer ..." means. Are the points supposed to be stops? Do we get stops which intersect with a provided point layer? Maybe this paragraph should be rephrased a bit.

Reply:
could you suggest a rephrasing ?